src/Tribe/Views/V2/functions/classes.php

```diff
@@ -94,3 +94,76 @@ function month_multiday_classes( $event, $day_date, $is_start_of_week, $today_da
 	 */
 	return apply_filters( 'tribe_events_views_v2_month_multiday_classes', $classes, $event, $day_date, $is_start_of_week, $today_date );
 }
+
+/**
+ * Used in the Month View days loop.
+ * Outputs classes for each day "cell".
+ *
+ * @since TBD
+ *
+ * @param array<mixed> $day          The current day data.
+ * @param string       $day_date     The current day date, in the `Y-m-d` format.
+ * @param DateTime     $request_date The request date for the view.
+ * @param string       $today_date   Today's date in the `Y-m-d` format.
+ *
+ * @return array<string> $day_classes The classes to add to the day "cell".
+ */
+function month_day_classes( $day, $day_date, $request_date, $today_date ) {
```

Comment:
All of the conditionals here could be simplified if we passed an array of the conditionals.

Reply:
Do you mean parameters?

Reply:
oh!